### ISSUE TITLE
Display a different temperature source for LibreHardwareMonitor

### ISSUE THREAD

**Issue by thewan056** [HelloWorld017/Fluent#1]
Hi. I have an AMD Bulldozer CPU, specifically the FX8120. I would like to display the CPU temperature as detected by the Motherboard sensor instead of the CPU sensor. How do I change to that sensor?

**Comment by HelloWorld017** [comment]
Unfortunately, for now, changing sensor is not supported.

But, you can manually change the source code at `@Resources/Scripts/OpenHardwareInfo.lua`.
You might want to change the 'cpu' of the line 29: `componentImg:match('cpu')`
I think 'mainboard' would work.

Sorry for the late response.

**Comment by thewan056** [comment]
It's alright. I just managed to try your suggestion. It doesn't work. The CPU temperature shows 0 when I changed it to 'mainboard'. 

I believe the reason why is that the temperature as detected by the motherboard is under the ITE chip. In LibreHardwareMonitor this chip is a child of the motherboard. So its Motherboard>ITE>Temperatures. Since your script is originally for the CPU>Temperatures, only looks for the first child maybe that is why it doesn't work. Unfortunately I am not good in programming/scripting stuff so I do not know how to modify your script to account for this additional child.

However, while waiting for your reply, I simply edited your temperature skin to use HWiNFO as a source which i was already using from an older rainmeter skin. Just copy paste here and there, no scripting. So now I am a happy user of your Fluent skin.
